api.planx.uk/types.ts

```diff
@@ -38,7 +38,7 @@ export interface Team {
 }
 
 export interface Passport {
-  data?: Record<string, any>;
+  data: Record<string, any>;
```

Comment:
This was required to match the `Passport` type in the `planx-document-templates` repo - going forward the plan would be to have a single definitive type in `planx-core`